### PR TITLE
Login feature - Frontend

### PR DESCRIPTION
**Pull request by lorance19**
Login and Sign Up feature can now send data to the backend. 

Update: Fully functional Login and Register and Logout feature. 
Login Feature now can work with backend and check if the user enters the right usename and password. 
Register Feature now can work with backend and check if the user is already register or not.
Now after user sign up, he will redirect to the root page with his name on the nav bar and logout button.

### CODE REVIEW

**Comment by willyspinner** [comment]
NOTE: we will deploy the backend separately, since it's blocking other stories and it is separable from frontend since we're using passport js auth (no more auth0).

**Comment by willyspinner** [comment]
UPDATE: backend merged to master & deployed (#29)

**Comment by akanksha003** [comment]
It seems that the Travis checks have failed. Could you take a look at those errors? 
I believe this is the problem: 
`Cannot find module: 'react-toastify'. Make sure this package is installed.` 
You can find more information here: [https://travis-ci.com/github/cs130-w21/19/builds/216773088](url)

**Review by akanksha003 — APPROVED** [review]
Alright, good job! This branch is ready for merging :)

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,8 @@
+<<<<<<< HEAD
+.DS_Store
+node_modules
+./client/.eslintcache
+=======
```
Comment (willyspinner) [inline .gitignore:5]: Please remove the git conflict marks here (`<<<<< HEAD` and `===========`).

File: .gitignore [hunk 2]
```diff
@@ -119,3 +124,4 @@ dist
 .elasticbeanstalk/*
 !.elasticbeanstalk/*.cfg.yml
 !.elasticbeanstalk/*.global.yml
+>>>>>>> 138988c83c16ac1b595760fa2559fe0e2fb956ee
```
Comment (willyspinner) [inline .gitignore:127]: here too

**Review by willyspinner — COMMENTED** [review]
This is ready to merge, but there's some git conflict marks (`<<<<<<` and `=======`) that you forgot to remove. I'll remove those now.

**Review by willyspinner — APPROVED** [review]
looks good now. Awesome job @lorance19 !

**Pull request merged**